src/__tests__/SubmitRemoval.test.tsx

```diff
@@ -0,0 +1,28 @@
+import {screen, render, fireEvent} from "@testing-library/react";
+import SubmitRemoval from "../components/SubmitRemoval";
+
+const testData = {
+    hideModal: ()=>{},
+    removeHandler: ()=>{}
+}
+
+describe('SubmmitRemovat test', ()=>{
```

Comment:
please provide a more descriptive message on what are you testing f.e. it should render are you sure model after btn click